### ISSUE TITLE
Reland the namespace change in `android/app/build.gradle`

### ISSUE THREAD

**Issue by navaronbracke** [navaronbracke/weforza#313]
Currently Flutter does not yet support the `namespace` field in the `android` section of the `build.gradle` file.

The fix is currently available on `master`, but we'll have to wait until it is merged into stable.

Remove the `package` attribute once the tooling is updated.
We should probably bump AGP & Gradle as well, once we land this change.
